src/javascript/binary/common_functions/currency_to_symbol.js

```diff
@@ -1,11 +1,36 @@
 function format_money(currency, amount) {
+    if(currency === 'JPY') { // remove decimal points for JPY and add comma.
+        amount = amount.replace(/\.\d+$/, '');
+        amount = amount.replace(/\B(?=(\d{3})+(?!\d))/g, ",");
```

Comment:
It's better to use a separate variable rather than mutate it, eg

```
var updatedAmount = amount.replace(/\.\d+$/, '').replace(/\B(?=(\d{3})+(?!\d))/g, ",")
```